src/dmd/root/stringtable.d

```diff
@@ -70,48 +82,35 @@ pure:
 struct StringTable
 {
 private:
-    StringEntry* table;
-    size_t tabledim;
-    ubyte** pools;
-    size_t npools;
+    StringEntry[] table;
+    ubyte*[] pools;
     size_t nfill;
     size_t count;
     size_t countTrigger;   // amount which will trigger growing the table
 
-nothrow:
 public:
-    void _init(size_t size = 0) nothrow pure
+nothrow:
```

Comment:
Just a nit to avoid code churn.  Can you move this back to appear above the `public:` specifier?

Reply:
I actually agree with @dkorpel that `public` should be before `nothrow` as this marks the separation between private/public.